src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -724,5 +724,13 @@ public class AgentKnobs
             "Show warning message on the OS which is not supported by .NET 8",
             new PipelineFeatureSource("Net8UnsupportedOsWarning"),
             new BuiltInDefaultKnobSource("true"));
+
+        public static readonly Knob UsePSScriptWrapper = new Knob(
+            nameof(UsePSScriptWrapper),
+            "Use PowerShell script wrapper to handle PowerShell ConstrainedLanguage mode.",
+            new RuntimeKnobSource("USE_PS_SCRIPT_WRAPPER"),
+            new EnvironmentKnobSource("USE_PS_SCRIPT_WRAPPER"),
+            new PipelineFeatureSource("UsePSScriptWrapper"),
```

Comment:
Do we expecting the user will toggle this feature manually?

Reply:
Removed unnecessary ones